### PR TITLE
Update hosted pade.doap

### PR DESCRIPTION
**Pull request by deleolajide**
Adding more information about XEPs inherited from ConverseJS and XEPs from additional plugins used.

### CODE REVIEW

File: static/hosted-doap/pade.doap [hunk 1]
```diff
@@ -5,9 +5,319 @@
          >
```
Comment (wurstsalat3000) [inline static/hosted-doap/pade.doap:5]: Please add ` xmlns:schema="https://schema.org/"` for the logo element.

**Comment by wurstsalat3000** [comment]
I would prefer to have the DOAP file hosted at Padé's repository, and keep xmpp.org's hosted DOAP files to a set of basic DOAP files. But if this doesn't work for you, hosting it in xmpp.org's repository should be fine.

**Comment by deleolajide** [comment]
I have it hosted at https://igniterealtime.github.io/pade/pade.doap and https://github.com/igniterealtime/pade/blob/master/docs/pade.doap

File: static/hosted-doap/pade.doap (hunk 1 above)
Comment (deleolajide) [reply static/hosted-doap/pade.doap:5]: Done

**Comment by wurstsalat3000** [comment]
Alright! I updated Pàdé's DOAP URL in a91efc6923c25c566db56723a5b5934958a03a24 and removed the hosted DOAP file in a48118993b547eb8b225802c2df9432867042d1c

So now updating the DOAP file at Pàdé's repository will be reflected on xmpp.org 🎉

**Comment by deleolajide** [comment]
Thank you 😊

**Pull request closed without merging**